frappe/public/js/frappe/views/communication.js

```diff
@@ -153,7 +153,14 @@ frappe.views.CommunicationComposer = class {
 				fieldname: "sender",
 				options: email_accounts.map(function(e) {
 					return e.email_id;
-				})
+				}),
+				change: async () => {
```

Comment:
Why does change event trigger twice? 🤔 I see that it has been an issue from a long time - https://discuss.erpnext.com/t/onchange-is-being-triggered-twice-in-page-field/52258